pkgs/borgbackup/borg/pkg.yaml

```diff
@@ -0,0 +1,22 @@
+packages:
+  - name: borgbackup/borg@1.1.12
```

Comment:
🤔 1.1.12 isn't latest. I'm not sure why `cmdx s` can't get the latest version.

Reply:
IIRC, the release API does not return releases in any particular order and you need to page. If you're already pulling _all_ releases, I don't know.

There _is_ a dedicated endpoint to get the latest release, so it should always be possible to get that one at least.

Reply:
By default `cmdx s` gets all versions and sort them by semver.
So it's weird. I'll look into this when I have time, but the priority is low.